src/SpineCenterline.jl

```diff
@@ -9,13 +9,14 @@ Reconstruct the coil sensitivity map using the MRIReco.jl function and save it i
 # Arguments
 * `path_nifti::String` - path of the nifti file. The file must have .nii extension
 * `path_rep::String` - path of reference data in ISMRMRD format
+* `tresh::Float64` - masking threshold: increase for reduced mask size, decrease for extended mask size
 
 MRIReco reference: https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28792
 ISMRMRD reference: https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.26089
 """
-function ReconstructSaveMap(path_nifti::String, path_ref::String)
+function ReconstructSaveMap(path_nifti::String, path_ref::String, thresh = 0.13)
```

Comment:
did you change the ReconstructSaveMap call in the CI tests (or is there one? If not then we can add it in a separate PR)

Reply:
@alexjaffray Currently there's no test implemented for the ReconstructSaveMap function. We need to add it.